### ISSUE TITLE
4.3.0 introduced output that is restricted output auth_string

### ISSUE THREAD

**Issue by btbinaryjanitor** [terraform-google-modules/terraform-google-memorystore#84]
### TL;DR

Error: Output refers to sensitive values
│
│   on outputs.tf line 47:
│   47: output "auth_string" {
│
│ To reduce the risk of accidentally exporting sensitive data that was
│ intended to be only internal, Terraform requires that any root module
│ output containing sensitive data be explicitly marked as sensitive, to
│ confirm your intent.
│
│ If you do intend to export this data, annotate the output value as
│ sensitive by adding the following argument:
│     sensitive = true
╵
ERRO[0009] 1 error occurred:
        * exit status 1

### Expected behavior

Sensitive outputs should be set to sensitive or not be an output at all in root modules.  Version 4.2.0 Handled this correctly.  4.3.0 even has an issue when you set auth_enabled       = false

### Observed behavior

Error: Output refers to sensitive values
│
│   on outputs.tf line 47:
│   47: output "auth_string" {
│
│ To reduce the risk of accidentally exporting sensitive data that was
│ intended to be only internal, Terraform requires that any root module
│ output containing sensitive data be explicitly marked as sensitive, to
│ confirm your intent.
│
│ If you do intend to export this data, annotate the output value as
│ sensitive by adding the following argument:
│     sensitive = true
╵
ERRO[0009] 1 error occurred:
        * exit status 1

### Terraform Configuration

```hcl
terraform {
  source = "tfr:///terraform-google-modules/memorystore/google?version=4.3.0"
}

include "root" {
  path = find_in_parent_folders("ephpr_automate_root.hcl")
}

dependencies {
  paths = []
}

locals {
    common_vars = read_terragrunt_config(find_in_parent_folders("ephpr_automate_root.hcl"))
}

inputs = {
  region             = "xxxx"
  project            = "xxxx"
  enable_apis        = true
  auth_enabled       = false
  name               = "xxxx"
  authorized_network = "xxxx"
  tier               = "BASIC"
  memory_size_gb     = 4
  #location_id = 
  #alternative_location_id = 
  redis_version      = "REDIS_6_X"
  redis_configs      = { maxmemory-policy = "allkeys-lru" }
  display_name       = "Persistent Cache"
  reserverd_ip_range = ""
  connect_mode       = "DIRECT_PEERING"
  labels = {
    terraform = "true"
  }
}
```

### Terraform Version

```sh
Terraform v1.0.11
on linux_amd64
```

**Comment by btbinaryjanitor** [comment]
This module is not supported to be used directly by Terragrunt.  It is recommended to use a wrapper around this module.
